src/content/docs/dropins/product-details/index.mdx

```diff
@@ -1,34 +1,45 @@
 
-The Product Details Page allows you to showcase your products according to your brand's aesthetics and user experience goals. Dropin features include:
+## Use cases
+
+The PDP dropin provides a variety of fully-customizable controls to showcase your products according to your brand's aesthetics and build interactive experiences that engage customers. Use cases include:
 
 - **Image Carousel**: Display multiple product images in a carousel format, allowing users to view different angles and details of the product.
 - **Product Information**: Showcase detailed product information, including pricing, descriptions, and specifications.
 - **Product Variants**: Display different product variants, such as colors, sizes, and styles, allowing users to select the option that best fits their needs.
 - **Product Reviews**: Include user reviews and ratings to provide social proof and help users make informed purchasing decisions.
-- **Product Recommendations**: Suggest related products or accessories to encourage users to explore additional items.
 - **Customization Options**: Customize the appearance and behavior of the dropin to align with your brand's design aesthetic and user experience goals.
 
 ## Anatomy
 
-The Product Details Page consists of the following prerequisites and components:
+<Diagram caption="PDP slots">![PDP example](@images/slots/pdp-slots.webp)</Diagram>
+
+## Render product types
 
-<Diagram>![PDP Anatomy](@images/placeholder.webp)</Diagram>
+The PDP dropin supports rendering different product types configured in your Adobe Commerce instance by default using data provided by the [Catalog Service GraphQL API](https://developer.adobe.com/commerce/services/graphql/catalog-service/). Adobe Commerce supports seven product types, but the Catalog Service GraphQL API schema maps these to two types:
 
-## Examples
+- _Simple products_ are products that are defined with a single price and quantity. Catalog Service maps the simple, virtual, downloadable, and gift card product types to [`simpleProductViews`](https://developer.adobe.com/commerce/services/graphql/catalog-service/products/#return-details-about-a-simple-product).
+- _Complex products_ are comprised of multiple simple products. The component simple products can have different prices. A complex product can also be defined so that the shopper can specify the quantity of component simple products. Catalog Service maps the configurable, bundle, and grouped product types to [`complexProductViews`](https://developer.adobe.com/commerce/services/graphql/catalog-service/products/#return-details-about-a-complex-product).
```

Comment:
Should this also include the recently added bundle type? With the important note that the drop-in support has the same limitations as the CS API. So what's supported in the API is supported on the drop-in.

Reply:
@alandana, line 41 mentions bundle product types.

Are there specific limitations we should highlight? I don't see any limitations in the [CS API docs](https://developer.adobe.com/commerce/services/graphql/catalog-service/).